app/models/miq_policy.rb

```diff
@@ -369,6 +370,38 @@ def self.seed
     end
   end
 
+  def self.seed_fixtures
+    fixture_file = File.join(FIXTURE_DIR, "miq_policies.yml")
+    fixtures = File.exist?(fixture_file) ? YAML.load_file(fixture_file) : []
+
+    fixtures.each do |policy|
+      rec = seed_attributes(policy.except(:conditions))
+      seed_conditions(rec, policy[:conditions])
+    end
+  end
+
+  def self.seed_attributes(policy)
```

Comment:
We're really seeding_policies or seed_policies_from_attributes here right?  Maybe also change the parameter to policy_attributes or policy_fixture or something so we know it's not a MiqPolicy object.